examples/dht_sensor/FreeRTOSConfig.h

```diff
@@ -0,0 +1,15 @@
+/* Blink FreeRTOSConfig overrides.
```

Comment:
You can delete this config file in order to get the default FreeRTOSConfig for the example, you probably don't need to override anything here.